docs/Realtime_e2e.txt

```diff
@@ -0,0 +1,19 @@
+
+               Device/Producer                                    Service                               KVS Consumer Client
+
++-------------+-------------+--------------------+            +----------------+            +--------------------+-------------+-------------+
+|             |             | HTTP Client  cFDA  |            |                |            | Parser        cFDA |             |             |
```

Comment:
What is cFDA and cFF9?

Reply:
Oh, these are the source files for drawing the PNG diagrams. These indicate the color of the object to draw